### PR TITLE
update item group file location

### PR DESCRIPTION
**Pull request by sader1992**
* **Addressed Issue(s)**: 

* **Server Mode**: 
both

* **Description of Pull Request**: 
just update the file location

### CODE REVIEW

**Comment by sader1992** [comment]
i remember that there was #define MAX_ITEMGROUP option in the source files but i did't find it :x

**Comment by Jeybla** [comment]
@sader1992 7f4a7ba26f8cfd8f929ddc2402f774c7b7294c82 Itemgroup has been changed to DBMap in that commit

File: doc/item_group.txt [hunk 1]
```diff
@@ -40,6 +40,8 @@ The table below explains which fields are accessed in each.
 
 GroupID: See the "Item Group ID" section in 'src/map/itemdb.h'.
          Supports IG_* constants.
+	and "Item Group ID" section in 'src/map/script_constants.h'.
+	 Supports export_constant(IG_* constants.
```
Comment (aleos89) [inline doc/item_group.txt]:
```
         See the "item groups" section in 'src/map/script_constants.h".
         Supports export_constant(IG_*) constants.
```

Comment (sader1992) [reply doc/item_group.txt]: do you mean remove the first line ? (See the "Item Group ID" section in 'src/map/itemdb.h'.)
if not i think for the second line it's better to start it with (and) so they know that there is two files must be edited to add new IG

**Pull request merged**